src/plopp/graphics/colormapper.py

```diff
@@ -114,11 +114,18 @@ def autoscale(self, data):
         elif vmax.value > self.vmax:
             self.vmax = vmax.value
 
-    def _set_children_colors(self):
-        for child in self.children.values():
-            child.set_colors(self.rgba(child.data))
+    def _set_children_colors(self, key=None):
+        """
+        Update the colors of all the children apart from the one that triggered the
+        update, as those get updated by the figure.
+        """
+        keys = set(self.children.keys())
+        if key is not None:
+            keys -= set([key])
```

Comment:
```suggestion
        keys = set(self.children.keys()) - {key}
```
no need to check for `None` unless that is a valid key (?)